reports/summary-2020.md

```diff
@@ -0,0 +1,819 @@
+    - working on mitigating the impact of removing beta APIs in 1.22
+    - server-side-apply to stable
+    - server-side-apply client
+    - optionally skip backend TLS verification
+    - namespace labels
+    - CRD and admission webhook v1beta1 API removal: reminder on kubernetes-dev.
+    - Immutable fields API
+    - API unions
+    - warnings to stable
+    - apiserver network proxy to beta
+    - priority and fairness to stable
+- [SIG Apps](https://git.k8s.io/community/sig-apps/annual-report-2021.md#current-initiatives-and-project-health)
+    - No report
+- [SIG Architecture](https://git.k8s.io/community/sig-architecture/annual-report-2021.md#current-initiatives-and-project-health)
+    - Increased coverage of stable endpoints by conformance tests
+    - Coordinating dependency updates across projects
+    - Production Readiness Review process was made mandatory in 1.21, improving 
+    scalability, supportability, monitoring, and correct feature enablement
+    - Set up cross-project policies to move features towards stable ([conformance without beta](https://git.k8s.io/enhancements/keps/sig-architecture/1333-conformance-without-beta), [preventing "permabeta"](https://git.k8s.io/enhancements/keps/sig-architecture/1635-prevent-permabeta))
+    - Enhancements subproject is working with sig-release to assist SIGs in 
+    taking greater ownership of their KEPs during the release cycle
+- [SIG Auth](https://git.k8s.io/community/sig-auth/annual-report-2021.md#current-initiatives-and-project-health)
+    - BoundServiceAccountToken
+    - CSR v1
+    - Token Request / bound SA token admission
+    - client-go auth plugins
+    - external kubelet credential providers
+    - New features in Secrets Store CSI driver
+    - Pod Security Policy Replacement
+    - and several other KEPs going to stable on the report
+- [SIG Autoscaling](https://git.k8s.io/community/sig-autoscaling/annual-report-2021.md#current-initiatives-and-project-health)
+    - No report
+- [SIG CLI](https://git.k8s.io/community/sig-cli/annual-report-2021.md#current-initiatives-and-project-health)
+    - Moving kubectl package code to staging
+    - Our multi-year effort to split out of the main kubernetes repository.
+    - kubectl debug (beta)
+    - Several smaller efforts to unify code across all the commands, and 
+    removing technical debt
+- [SIG Cloud Provider](https://git.k8s.io/community/sig-cloud-provider/annual-report-2021.md#current-initiatives-and-project-health)
+    - Feature: implement the BackendManager list
+    - Fix flag passing in CCM
+    - Extending Apiserver Network Proxy to handle traffic originated from Node 
+    network
+- [SIG Cluster Lifecycle](https://git.k8s.io/community/sig-cluster-lifecycle/annual-report-2021.md#current-initiatives-and-project-health)
+    - Standard for communicating a local registry
+    - several KEPs in a separate KEP process https://github.com/kubernetes-sigs/cluster-api/tree/master/docs/proposals
```

Comment:
```suggestion
- [SIG Cluster Lifecycle](https://git.k8s.io/community/sig-cluster-lifecycle/annual-report-2021.md#current-initiatives-and-project-health)
    - Standard for communicating a local registry
    - Several KEPs in a separate KEP process: https://github.com/kubernetes-sigs/cluster-api/tree/master/docs/proposals
    
```

Reply:
.... & ecetera for the rest of this list:

1) Please capitalize each list item (unless referring to a code artifact which must be lowercased) 
2) please add an enter (carriage return) at the end of each SIG sublist – this makes it far easier to read and prevents weird markdown rendering.